VotingApplication/VotingApplication.Web.Tests/E2E/BasicPollTests.cs

```diff
@@ -179,22 +87,26 @@ public void AfterVoting_VoteIsRemembered()
                     using (var context = new TestVotingContext())
                     {
                         CreatePoll(context);
-                        GoToUrl(driver, PollUrl);
+                        GoToUrl(driver, PollVoteUrl);
 
-                        IWebElement voteButtons = FindElementById(driver, "vote-button");
-                        voteButtons.Click();
+                        IReadOnlyCollection<IWebElement> choices = driver.FindElements(NgBy.Binding("choice.Name"));
+                        choices.First().Click();
+
+                        IWebElement voteButton = FindElementById(driver, "vote-button");
+                        voteButton.Click();
 
                         NavigateBackToVotePage(driver);
 
-                        IWebElement selectedChoice = driver.FindElement(By.CssSelector(".selected-choice"));
+                        IWebElement selectedChoice = driver.FindElement(By.CssSelector(".md-ripple-visible"));
```

Comment:
Just translating the existing tests so they pass under the new design, but I agree this isn't particularly how we want to be doing it. We could check the model, but then we're not testing the UI display.